orchestrator/main_loops.go

```diff
@@ -57,18 +57,25 @@ var estimatedGasCosts = []int64{
 func (p *gravityOrchestrator) Start(ctx context.Context) error {
 	var pg loops.ParanoidGroup
 
-	pg.Go(func() error {
-		return p.EthOracleMainLoop(ctx)
-	})
+	if p.ethMergePause {
+		p.logger.Warn().Msg("Eth claims are disabled. This is OK if we are during the ETH merge pause period.")
+		pg.Go(func() error {
+			return p.EthOracleMainLoop(ctx)
+		})
+	}
 	pg.Go(func() error {
 		return p.BatchRequesterLoop(ctx)
 	})
 	pg.Go(func() error {
 		return p.EthSignerMainLoop(ctx)
 	})
```

Comment:
we need to run this with pause==true?

Reply:
Sorry, let me go over it again, I fucked up during the stashing and applying